src/wp-includes/formatting.php

```diff
@@ -4878,7 +4878,7 @@ function sanitize_option( $option, $value ) {
 			break;
 
 		case 'moderation_keys':
-		case 'blacklist_keys':
+		case 'blocklist_keys':
```

Comment:
Is there a danger of a plugin calling `update_option()` and `get_option` on the deprecated option name themselves, and thus opening themselves up to a security issue of some kind?

Reply:
Good question! I have that in the checklist in the PR description. I plan to add a filter automatically changing any option name that is changed here to the new name, and possibly triggering a `_doing_it_wrong()`. I want to explore if this had been done before. I couldn't find any signs of something similar on a brief glance.

Reply:
Instead of tackling this in Core, I am going to include this in a dev note and work with the plugin team to do some outreach (see also my [previous comment](https://github.com/WordPress/wordpress-develop/pull/341#issuecomment-646837774)).